awx/main/expect/isolated_manager.py

```diff
@@ -319,58 +257,51 @@ def health_check(cls, instance_qs, awx_application_version):
          - clean up orphaned private files
         Performs save on each instance to update its capacity.
         '''
-        hostname_string = ''
-        for instance in instance_qs:
-            hostname_string += '{},'.format(instance.hostname)
-        args = cls._build_args('heartbeat_isolated.yml', hostname_string)
-        args.extend(['--forks', str(len(instance_qs))])
-        env = cls._base_management_env()
+        # TODO: runner doesn't have a --forks arg
+        #args.extend(['--forks', str(len(instance_qs))])
 
         try:
-            facts_path = tempfile.mkdtemp()
-            env['ANSIBLE_CACHE_PLUGIN'] = 'jsonfile'
-            env['ANSIBLE_CACHE_PLUGIN_CONNECTION'] = facts_path
-
-            buff = StringIO()
-            timeout = max(60, 2 * settings.AWX_ISOLATED_CONNECTION_TIMEOUT)
-            status, rc = IsolatedManager.run_pexpect(
-                args, cls.awx_playbook_path(), env, buff,
-                idle_timeout=timeout, job_timeout=timeout,
-                pexpect_timeout=5
+            private_data_dir = tempfile.mkdtemp(
+                prefix='awx_iso_heartbeat_',
```

Comment:
good, this makes it easier to find. Any other contextual data we might want to include would be helpful.